src/components/HomeNavBar.tsx

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import MenuIcon from '@material-ui/icons/Menu';
+import AccountCircleIcon from '@material-ui/icons/AccountCircle';
+import Drawer from '@material-ui/core/Drawer';
+import List from '@material-ui/core/List';
+import Divider from '@material-ui/core/Divider';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemIcon from '@material-ui/core/ListItemIcon';
+import ListItemText from '@material-ui/core/ListItemText';
+import PersonIcon from '@material-ui/icons/Person';
+import BuildIcon from '@material-ui/icons/Build';
+import ShoppingCartIcon from '@material-ui/icons/ShoppingCart';
+import ReportProblemIcon from '@material-ui/icons/ReportProblem';
+import LocationOnIcon from '@material-ui/icons/LocationOn';
+import WbSunnyIcon from '@material-ui/icons/WbSunny';
+import { Root, Bar, MenuButton, AccountButton, ListStyle, ToolBar, Typograph } from '../styles/TopNavStyles';
+
+type Anchor = 'left';
```

Comment:
Because the drawer will only ever be opening up on the left side of the screen (and won't change position), it's not necessary to specify this. An `isOpen` state boolean would probably suffice.